### ISSUE TITLE
Note Preview doesn't load

### ISSUE THREAD

**Issue by OldStarchy** [dendronhq/dendron#2214]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [x] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

After installing dendron and following the prompts to get started, the `tutorial.md` opens and a tab `Note Preview` opens next to it and just has the text "Loading...". After that nothing happens, and the preview doesn't load.

## To Reproduce

1. Install dendron
2. Follow the prompts to get started
4. See Note Preview doesn't load

## Expected behavior

I am supposed to see something but I don't know what to expect yet as I haven't seen it working

## Screenshots

![image](https://user-images.githubusercontent.com/15243523/150243513-d0c2e118-dd75-4494-ae2b-d7d80b8c7a11.png)

## Desktop (please complete the following information)

- Version: 1.64.0-insider (user setup)
- Commit: 7ae7f9d40b82a0f1a49d8ad180d27e16812ad95e
- Date: 2022-01-18T05:16:47.915Z
- Electron: 13.5.2
- Chromium: 91.0.4472.164
- Node.js: 14.16.0
- V8: 9.1.269.39-electron.0
- OS: Windows_NT x64 10.0.19043

## Additional context

I've tried setting up dendron locally, via WSL, and via remote SSH on two different servers (ubuntu 20.04 and retropie linux).

This is my second time installing dendron, I had the same problem a few months ago and decided to come back to it later.

## Dendron Log file

<details>

```
{"level":30,"time":1642640189051,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"Logger:configure","msg":"exit","logLevel":"info"}
{"level":30,"time":1642640190023,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"_activate","stage":"prod","isDebug":false,"logLevel":"info","logPath":"/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron","extensionPath":"/home/pi/.vscode-server-insiders/extensions/dendron.dendron-0.78.0","extensionUri":"/home/pi/.vscode-server-insiders/extensions/dendron.dendron-0.78.0","workspaceFile":"/home/pi/Dendron/dendron.code-workspace","workspaceFolders":["/home/pi/Dendron/vault"]}
{"level":30,"time":1642640190032,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"No cache path for Segment specified. Failed event uploads will not be retried."}
{"level":30,"time":1642640190034,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"user telemetry setting: enabled by config"}
{"level":30,"time":1642640190035,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"telemetry initializing"}
{"level":30,"time":1642640190035,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"using existing id"}
{"level":30,"time":1642640190058,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"DendronExtension","msg":"initialized"}
{"level":30,"time":1642640190076,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"_activate","msg":"initializeWorkspace","wsType":"CODE","currentVersion":"0.78.0","previousWorkspaceVersion":"0.0.0","previousGlobalVersion":"0.78.0","extensionInstallStatus":"NO_CHANGE"}
{"level":30,"time":1642640190140,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"user telemetry setting: enabled by config"}
{"level":30,"time":1642640190141,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"telemetry initializing"}
{"level":30,"time":1642640190141,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"using existing id"}
{"level":30,"time":1642640190142,"pid":8675,"hostname":"retropie","name":"dendron","msg":"Telemetry is disabled? false"}
{"level":30,"time":1642640190142,"pid":8675,"hostname":"retropie","name":"dendron","msg":"Segment Residual Cache Path is at /home/pi/.vscode-server-insiders/data/User/globalStorage/dendron.dendron/segmentresidualcache.log"}
{"level":30,"time":1642640190144,"pid":8675,"hostname":"retropie","name":"SegmentClient","msg":"Attempting to flush residual segment data from file."}
{"level":30,"time":1642640190342,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"_activate","workspaceInstallStatus":"INITIAL_INSTALL","currentVersion":"0.78.0","previousWorkspaceVersion":"0.0.0","previousGlobalVersion":"0.78.0","platform":"linux","extensions":[{"id":"dendron.dendron-paste-image","version":"1.1.0","active":false},{"id":"dendron.dendron-markdown-shortcuts"},{"id":"redhat.vscode-yaml"}],"vaults":[{"fsPath":"vault"}]}
{"level":30,"time":1642640190365,"pid":8675,"hostname":"retropie","name":"execServer","state":"enter"}
{"level":30,"time":1642640190397,"pid":8675,"hostname":"retropie","name":"execServer","state":"post:exec.node"}
{"level":30,"time":1642640195550,"pid":8718,"hostname":"retropie","name":"SegmentClient","msg":"No cache path for Segment specified. Failed event uploads will not be retried."}
{"level":30,"time":1642640195552,"pid":8718,"hostname":"retropie","name":"SegmentClient","msg":"user telemetry setting: enabled by config"}
{"level":30,"time":1642640195554,"pid":8718,"hostname":"retropie","name":"SegmentClient","msg":"telemetry initializing"}
{"level":30,"time":1642640195555,"pid":8718,"hostname":"retropie","name":"SegmentClient","msg":"using existing id"}
{"level":30,"time":1642640197895,"pid":8675,"hostname":"retropie","name":"execServer","state":"message","message":"33613"}
{"level":30,"time":1642640197897,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WSUtils.handleServerProcess","msg":"subprocess running","pid":8718}
{"level":30,"time":1642640197899,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"startServerProcess","msg":"post-start-server","port":33613,"durationStartServer":7819}
{"level":30,"time":1642640197992,"pid":8675,"hostname":"retropie","name":"logger","ctx":"FileStorageV2","wsRoot":"/home/pi/Dendron","vaults":[{"fsPath":"vault"}],"level":"info"}
{"level":30,"time":1642640198005,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.reloadIndex:run","msg":"pre-execute"}
{"level":30,"time":1642640198007,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"ReloadIndex.execute","msg":"enter"}
{"level":30,"time":1642640198480,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":0,"noteRefsLength":0,"wikiLinksLength":0}
{"level":30,"time":1642640198711,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":6,"noteRefsLength":0,"wikiLinksLength":6}
{"level":30,"time":1642640198940,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":3,"noteRefsLength":0,"wikiLinksLength":3}
{"level":30,"time":1642640199215,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":2,"noteRefsLength":0,"wikiLinksLength":2}
{"level":30,"time":1642640200030,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":5,"noteRefsLength":0,"wikiLinksLength":5}
{"level":30,"time":1642640200183,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":3,"noteRefsLength":1,"wikiLinksLength":2}
{"level":30,"time":1642640200336,"pid":8718,"hostname":"retropie","name":"LinkUtils.getLinks","ctx":"getLinks","dlinksLength":1,"noteRefsLength":0,"wikiLinksLength":1}
{"level":30,"time":1642640200579,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"ReloadIndex.execute","durationEngineInit":2568}
{"level":30,"time":1642640200581,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"ReloadIndex.execute","msg":"exit"}
{"level":30,"time":1642640200582,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.reloadIndex:run","msg":"post-execute"}
{"level":30,"time":1642640200627,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"reloadWorkspace","msg":"post-ws.reloadWorkspace"}
{"level":30,"time":1642640200637,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"reloadWorkspace","msg":"exit"}
{"level":30,"time":1642640200638,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"postReloadWorkspace","msg":"no previous global version"}
{"level":30,"time":1642640200700,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.upgradeSettings:run","msg":"pre-execute"}
{"level":30,"time":1642640200702,"pid":8675,"hostname":"retropie","name":"UpgradeSettingsCommand","ctx":"Upgrade:execute"}
{"level":30,"time":1642640200823,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WorkspaceConfig:update","vscodeDir":"/home/pi/Dendron/vault/.vscode","snippetChanges":{}}
{"level":30,"time":1642640200824,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"Upgrade:execute","newConfig":{"extensions":{},"settings":{"add":{},"errors":{}},"snippetChanges":{}}}
{"level":30,"time":1642640200825,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"Upgrade:execute","badExtensions":[]}
{"level":30,"time":1642640200826,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.upgradeSettings:run","msg":"post-execute"}
{"level":30,"time":1642640200834,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"postReloadWorkspace","msg":"postUpgrade: new wsVersion"}
{"level":30,"time":1642640200910,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"postReloadWorkspace","msg":"exit"}
{"level":30,"time":1642640200912,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree: enter"}
{"level":30,"time":1642640200922,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"setupViews","msg":"init:treeViewV2"}
{"level":30,"time":1642640200931,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"setupBacklinkTreeView","msg":"init:backlinks"}
{"level":30,"time":1642640200949,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree: exit"}
{"level":30,"time":1642640200968,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"activateWorkspace","stage":"prod","msg":"enter"}
{"level":30,"time":1642640200992,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"togglePluginActiveContext","state":"togglePluginActiveContext: true"}
{"level":30,"time":1642640200993,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"_activate","msg":"fin startClient","durationReloadWorkspace":10866}
{"level":30,"time":1642640200995,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.78.0","previousExtensionVersion":"0.0.0"}
{"level":30,"time":1642640201811,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/Dendron/vault/tutorial.md"]}
{"level":30,"time":1642640201848,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640201851,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onFirstOpen","msg":"enter","fname":"tutorial"}
{"level":30,"time":1642640202161,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onFirstOpen","msg":"exit","fname":"tutorial"}
{"level":30,"time":1642640202756,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640203642,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640205097,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640226069,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":[]}
{"level":30,"time":1642640226074,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640275465,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/Dendron/vault/tutorial.md"]}
{"level":30,"time":1642640275571,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640275646,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640276096,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640277455,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":[]}
{"level":30,"time":1642640277457,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640277481,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/Dendron/vault/tutorial.md"]}
{"level":30,"time":1642640277534,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640277568,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640279095,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640289708,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"pre-execute"}
{"level":30,"time":1642640289751,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"post-execute"}
{"level":30,"time":1642640291155,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640292979,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"pre-execute"}
{"level":30,"time":1642640293114,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640293117,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"post-execute"}
{"level":30,"time":1642640293418,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640320674,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":[]}
{"level":30,"time":1642640320676,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640338427,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/Dendron/vault/tutorial.md"]}
{"level":30,"time":1642640338480,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640338521,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640339023,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640339788,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640341923,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"pre-execute"}
{"level":30,"time":1642640342083,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640342087,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"post-execute"}
{"level":30,"time":1642640342682,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640351974,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.dev.openLogs:run","msg":"pre-execute"}
{"level":30,"time":1642640351976,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"execute","opts":{}}
{"level":30,"time":1642640352057,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.dev.openLogs:run","msg":"post-execute"}
{"level":30,"time":1642640352299,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":[]}
{"level":30,"time":1642640352301,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640352361,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log"]}
{"level":30,"time":1642640352373,"pid":8675,"hostname":"retropie","name":"dendron","uri":"/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log","msg":"not in workspace"}
{"level":30,"time":1642640367483,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640370190,"pid":8675,"hostname":"retropie","name":"dendron","uri":"/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log","msg":"not in workspace"}
{"level":30,"time":1642640377153,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640377156,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":[]}
{"level":30,"time":1642640377165,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log"]}
{"level":30,"time":1642640377169,"pid":8675,"hostname":"retropie","name":"dendron","uri":"/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log","msg":"not in workspace"}
{"level":30,"time":1642640377204,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeVisibleTextEditors","editorPaths":["/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log","/home/pi/Dendron/vault/tutorial.md"]}
{"level":30,"time":1642640380871,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640380874,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640380906,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640381251,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640383468,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640384418,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640384970,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640388973,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640394801,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640395206,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640397329,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"pre-execute"}
{"level":30,"time":1642640397355,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"post-execute"}
{"level":30,"time":1642640403309,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640405344,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"pre-execute"}
{"level":30,"time":1642640405456,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640405464,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"dendron.showPreview:run","msg":"post-execute"}
{"level":30,"time":1642640406021,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640412323,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"undefined"}
{"level":30,"time":1642640414925,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640416317,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640587536,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640588774,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640599790,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640599818,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640600003,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
{"level":30,"time":1642640840989,"pid":8675,"hostname":"retropie","name":"dendron","uri":"/home/pi/.vscode-server-insiders/data/logs/20220120T005525/exthost2/dendron.dendron/dendron.log","msg":"not in workspace"}
{"level":30,"time":1642640841016,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"WindowWatcher:onDidChangeTextEditorVisibleRanges","editor":"/home/pi/Dendron/vault/tutorial.md"}
{"level":30,"time":1642640842168,"pid":8675,"hostname":"retropie","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":8,"diagnosticsLength":0}}
```

</details>

**Comment by hikchoi** [comment]
Thank you for the report.

We have an existing issue regarding the same issue with accessing a Dendron workspace over SSH.
https://github.com/dendronhq/dendron/issues/2010

We'll keep track of this and investigate how to fix this issue.

**Comment by Kazephil** [comment]
I also had this problem (which seems to be the same as #2316), on an Arch Linux system with VSCodium version 1.63.2. I also had the additional issue of the Calendar View not loading. Based on one of the comments in #2010 and some of the information in the Dendron documentation about running a local server for the Preview and Calendar View functions, I suspected some sort of connection problem even though my log didn't show a "connection refused" error.

Looking into it, I realized that I wasn't able to connect to localhost from a browser either, suggesting a problem with my system rather than with Dendron.

It turned out that my _hosts_ file was missing the entry to resolve localhost to 127.0.0.1. Fixing the _hosts_ file using the information under "Local hostname resolution" in the [ArchWiki](https://wiki.archlinux.org/title/Network_configuration#Set_the_hostname) fixed the problem for me.

Hope this proves useful.
